### PR TITLE
Partially address font issues

### PR DESCRIPTION
**Pull request by BSG-75**
Windows by itself can handle font fallback, but for OpenSage which uses cross platform libraries, it could be quite hard, especially when our libraries cannot handle non-English font family names yet.
This pull request will:
- [X] Allow reading those "9x" ini files (which contain stuffs in locale-specific encoding) mentioned in https://github.com/OpenSAGE/OpenSAGE/issues/405
- [X] Use a user-configurable JSON file (Which will be `UserSettingsFiles/FontFallbackSettings.json`) as a temporary solution to handle font fallbacks, which for now should fix Chinese characters rendering in https://github.com/OpenSAGE/OpenSAGE/issues/398

### CODE REVIEW

File: src/OpenSage.Game/Content/Translation/Providers/CsfTranslationProvider.cs [hunk 1]
```diff
@@ -21,7 +21,7 @@ private enum LanguageGenerals
             [Display("ja-JP")] Japanese,
             [Display("en-US")] Jabber,
             [Display("ko-KR")] Korean,
-            [Display("zh-CN")] ChineseSimplified,
+            [Display("zh-HK")] ChineseTraditional,
```
Comment (feliwir) [inline src/OpenSage.Game/Content/Translation/Providers/CsfTranslationProvider.cs]: Are you sure that the other one isn't used at all / can be replaced?

File: src/OpenSage.Game/OpenSage.Game.csproj [hunk 2]
```diff
@@ -39,7 +39,7 @@
   <ItemGroup>
     <ShaderFiles Include="Assets\Shaders\*.vert;Assets\Shaders\*.frag" />
     <UpToDateCheckInput Include="@(ShaderFiles)" />
-    
+
```
Comment (feliwir) [inline src/OpenSage.Game/OpenSage.Game.csproj]: Those newlines probably can be removed again

File: src/OpenSage.Game/UserSettingsFiles/FontFallbackSettings.json [hunk 3]
```diff
@@ -0,0 +1,26 @@
+﻿{
```
Comment (feliwir) [inline src/OpenSage.Game/UserSettingsFiles/FontFallbackSettings.json]: I'd probably put this file to Content/Translation or Content/Fonts

File: src/OpenSage.Game/Content/Translation/Providers/CsfTranslationProvider.cs (hunk 1 above)
Comment (BSG-75) [reply src/OpenSage.Game/Content/Translation/Providers/CsfTranslationProvider.cs]: Even if the other one (simplified Chinese) exist, we don't know it's language code in Generals, so... for now I think it doesn't have a place in this enum, so it should just be replaced

File: src/OpenSage.Game/UserSettingsFiles/FontFallbackSettings.json (hunk 3 above)
Comment (BSG-75) [reply src/OpenSage.Game/UserSettingsFiles/FontFallbackSettings.json]: All right, will change it to Content/Fonts

File: src/OpenSage.Game/OpenSage.Game.csproj (hunk 2 above)
Comment (BSG-75) [reply src/OpenSage.Game/OpenSage.Game.csproj]: Just checked the project file, it always had new lines, should we remove all extra new lines, or you meant restore those removed spaces so they don't appear in the git diff?

Comment (feliwir) [reply src/OpenSage.Game/OpenSage.Game.csproj]: Ah you just removed spaces. Didn't look correctly then :) Just keep it as is.

File: src/OpenSage.Game/Content/Translation/Providers/CsfTranslationProvider.cs (hunk 1 above)
Comment (feliwir) [reply src/OpenSage.Game/Content/Translation/Providers/CsfTranslationProvider.cs]: Alright

File: src/OpenSage.Game/OpenSage.Game.csproj (hunk 2 above)
Comment (BSG-75) [reply src/OpenSage.Game/OpenSage.Game.csproj]: So I'll just keep the "remove space" change?

Comment (feliwir) [reply src/OpenSage.Game/OpenSage.Game.csproj]: Yes

**Pull request merged**